task4.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+import heapq as hq
+import pickle
+
+def minimun_index_list(list, k):
+    return hq.nsmallest(k, range(len(list)), list.__getitem__)
```

Comment:
please replace this with `list.argsort()[:k]` it does the same, its faster and cleaner :)

Reply:
Also choose a more descriptive name instead of `list`